proto/catalog.proto

```diff
@@ -85,7 +85,8 @@ message StreamSourceInfo {
   map<string, string> format_encode_options = 14;
 
   // Handle the source relies on any sceret. The key is the propertity name and the value is the secret id and type.
-  map<string, secret.SecretRef> secret_ref = 16;
+  // For format and encode options.
+  map<string, secret.SecretRef> format_encode_secret_refs = 16;
```

Comment:
Is this breaking? Will empty field present in the json encoding?

Reply:
No, it's not breaking. `StreamSourceInfo` is stored as `blob` in sql backend, It has the same compatibility as proto.